File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
a, b = map(int,input().split())

max_ = 0
for i in range(1, min(a,b)+1):
    if a % i == 0 and b % i == 0:
        max_ = i
min_ = a*b//max_ 
print(max_)
print(min_)

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.swift
func solution(_ n:Int, _ m:Int) -> [Int] {
    var answer = [Int]()
    var max = min(n, m)
    
    for i in 1...max {
        if n % i == 0 && m % i == 0 {
            max = i
        }
    }
    answer.append(max)
    answer.append(n*m/max)
    return answer
}